From a12b6af9954d6842742981768262f641abd9cf8e Mon Sep 17 00:00:00 2001
From: Shahar Levi <shahar_levi@ti.com>
Date: Mon, 28 May 2012 17:03:41 +0300
Subject: [PATCH 1/1] Remove copying of compat.ko

The compat.ko file is been copy via nlcp-compat-ko project
remove the copy from here

Signed-off-by: Shahar Levi <shahar_levi@ti.com>
---
 Android.mk |    8 --------
 1 files changed, 0 insertions(+), 8 deletions(-)

diff --git a/Android.mk b/Android.mk
index 4072322..4f1bdfe 100644
--- a/Android.mk
+++ b/Android.mk
@@ -2,14 +2,6 @@ local_target_dir := $(TARGET_OUT)/lib/modules
 LOCAL_PATH:= $(call my-dir)
 
 include $(CLEAR_VARS)
-LOCAL_MODULE := compat.ko
-LOCAL_MODULE_TAGS := debug
-LOCAL_MODULE_PATH := $(local_target_dir)
-LOCAL_SRC_FILES := $(LOCAL_MODULE)
-LOCAL_MODULE_CLASS := ETC
-include $(BUILD_PREBUILT)
-
-include $(CLEAR_VARS)
 LOCAL_MODULE := mac80211.ko
 LOCAL_MODULE_TAGS := debug
 LOCAL_MODULE_PATH := $(local_target_dir)
-- 
1.7.0.4

